frontend/src/components/@common/Tags/index.tsx

```diff
@@ -0,0 +1,3 @@
+import Tags from '~/components/@common/Tags/Tags';
```

Comment:
이것도 위의 상대경로 리뷰와 동일합니다~!

```suggestion
import Tags from './Tags';
```

Reply:
> 상대 경로 사용과 절대 경로 사용 중 하나로 통일하는 것이 좋다고 판단하여 절대경로로 모두 사용했습니다. 어떻게 생각하시나여?? 👍